include/pinocchio/bindings/python/spatial/inertia.hpp

```diff
@@ -295,6 +312,251 @@ namespace pinocchio
+      typedef Eigen::Matrix<Scalar, Eigen::Dynamic, 1, Options> VectorXs;
+      typedef Eigen::Matrix<Scalar, Eigen::Dynamic, Eigen::Dynamic, Options> MatrixXs;
+
+    public:
+      template<class PyClass>
+      void visit(PyClass & cl) const
+      {
+        cl.def(bp::init<const VectorXs &>((bp::arg("self"), bp::arg("log_cholesky_parameters"))))
+          .def(bp::init<const LogCholeskyParameters &>(
+            (bp::arg("self"), bp::arg("clone")), "Copy constructor"))
+
+          .add_property(
+            "parameters", &LogCholeskyParametersPythonVisitor::getParameters,
+            &LogCholeskyParametersPythonVisitor::setParameters, "Log Cholesky parameters.")
+
+          .def(
+            "toDynamicParameters", &LogCholeskyParametersPythonVisitor::toDynamicParameters_proxy,
+            bp::arg("self"), "Returns the dynamic parameters representation.")
+          .def(
+            "toPseudoInertia", &LogCholeskyParameters::toPseudoInertia, bp::arg("self"),
+            "Returns the Pseudo Inertia representation.")
+          .def(
+            "toInertia", &LogCholeskyParameters::toInertia, bp::arg("self"),
+            "Returns the Inertia representation.")
+          .def(
+            "calculateJacobian", &LogCholeskyParametersPythonVisitor::calculateJacobian_proxy,
+            bp::arg("self"), "Calculates the Jacobian of the log Cholesky parameters.")
+
+          .def("__array__", &LogCholeskyParameters::toDynamicParameters)
+          .def("__array__", &__array__)
+#ifndef PINOCCHIO_PYTHON_NO_SERIALIZATION
+          .def_pickle(Pickle())
+#endif
+          ;
+        PINOCCHIO_COMPILER_DIAGNOSTIC_POP
+      }
+
+      static VectorXs getParameters(const LogCholeskyParameters & self)
+      {
+        return self.parameters;
+      }
+      static void setParameters(LogCholeskyParameters & self, const Vector10 & parameters)
+      {
+        self.parameters = parameters;
+      }
+
+      static VectorXs toDynamicParameters_proxy(const LogCholeskyParameters & self)
+      {
+        return self.toDynamicParameters();
+      }
+
+      static MatrixXs calculateJacobian_proxy(const LogCholeskyParameters & self)
+      {
+        return self.calculateJacobian();
+      }
+
+      static void expose()
+      {
+#if PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION == 6 && EIGENPY_VERSION_AT_LEAST(2, 9, 0)
+        typedef PINOCCHIO_SHARED_PTR_HOLDER_TYPE(LogCholeskyParameters) HolderType;
+#else
+        typedef ::boost::python::detail::not_specified HolderType;
+#endif
+        bp::class_<LogCholeskyParameters, HolderType>(
+          "LogCholeskyParameters",
+          "This class represents log Cholesky parameters.\n\n"
+          "Supported operations ...",
+          bp::no_init)
+          .def(LogCholeskyParametersPythonVisitor<LogCholeskyParameters>())
+          .def(CopyableVisitor<LogCholeskyParameters>())
+          .def(PrintableVisitor<LogCholeskyParameters>());
+        // .def(CastVisitor<LogCholeskyParameters>())
```

Comment:
__array__ should have the following signature static Matrix4 __array__(const PseudoInertia & self, bp::object, bp::object)